src/Cake.Svn/Info/SvnInfoSettings.cs

```diff
@@ -10,16 +10,16 @@ public sealed class SvnInfoSettings : SvnRemoteSettings
         /// </summary>
         public SvnInfoSettings()
         {
-            Depth = SvnDepth.Infinity;
```

Comment:
What's the reason for this change?

Reply:
@pascalberger Infinity is not the default on SvnInfoArgs. And if we get information from a file we only get one result for this file. Same should be for a directory. If we want more information about directory and the subdirectories/files inside we should explicit demand this with changing the new default to e.g. Infinity.